cli/azd/resources/scaffold/templates/ai-project.bicept

```diff
@@ -83,11 +98,7 @@ resource {{bicepName .Name}}{{bicepName .Version}}Deploy 'Microsoft.CognitiveSer
     name: 'S0'
   }
   kind: 'AIServices'
-  properties: {
-    apiProperties: {
-      statisticsEnabled: false
-    }
-  }
+  properties: {}
```

Comment:
`statisticsEnabled` is only used for Bing Search and it seems the Cognitive Services RP now has stricter validation on this, causing create requests to fail when this is set.

![image](https://github.com/user-attachments/assets/ff706987-0b11-4c63-acf0-29c236ef819b)